packages/dd-trace/src/config.js

```diff
@@ -141,6 +141,17 @@ class Config {
       false
     )
 
+    const DD_TRACE_STATS_COMPUTATION_ENABLED = coalesce(
+      options.stats,
+      process.env.DD_TRACE_STATS_COMPUTATION_ENABLED,
+      false
+    )
+    const DD_TRACE_STATS_WRITER_INTERVAL = coalesce(
```

Comment:
Not sure, but the other languages support this too, so I just went for parity.

Reply:
I think we should check if this is mandatory, because it removes flexibility if we want for example to change how we report data to the agent later on and batch things etc. Same applies to the current `flushInterval` which we should eventually deprecate.

Reply:
Looks like it's just python that has it, so I'll just remove it.